spec/views/users/users_index_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+RSpec.describe 'User', type: :feature, js: false do
+  before :each do
+    @user = User.create(name: 'Naledi Dikgale', photo: 'https://e0.pxfuel.com/wallpapers/820/800/desktop-wallpaper-cute-black-girls-black-cartoon-characters-thumbnail.jpg',
+                        bio: 'Undisputed Queen of DIY')
+    @first_post = Post.create(author: @user, title: 'Naledi', text: 'How to make a table')
+    @comment = Comment.create(post: @first_post, author: @user, text: 'Nice post!')
+
+    visit "/users/#{@user.id}/posts"
+  end
```

Comment:
- [ ] The tests written in this file suppose to be for `users show` page according to the file name I think, the `url` added is pointing at the` post homepage` not `users show' page`. 👍 
- [ ] Kindly implement all the tests for user_show page as indicated in the requirement. 👍